ddtrace/internal/telemetry/__init__.py

```diff
@@ -0,0 +1,38 @@
+from ...internal import forksafe
+from .writer import TelemetryWriter
+
+
+TELEMETRY_WRITER = TelemetryWriter()
+
+__all__ = ["enable", "disable", "add_integration"]
+
+
+def add_integration(integration_name, auto_enabled):
+    # type: (str, bool) -> None
+    TELEMETRY_WRITER.add_integration(integration_name, auto_enabled)
+
+
+def _restart():
+    # type: () -> None
+    disable()
+    enable()
+
+
+def disable():
+    # type: () -> None
+    forksafe.unregister(_restart)
+
+    TELEMETRY_WRITER.stop()
+    TelemetryWriter.enabled = False
+
+    TELEMETRY_WRITER.join()
+
+
+def enable():
+    # type: () -> None
+    TELEMETRY_WRITER.start()
+    TelemetryWriter.enabled = True
+
+    forksafe.register(_restart)
+
+    TELEMETRY_WRITER.app_started_event()
```

Comment:
TODO: Verify what happens if we call `disable()` twice or `enable()` twice, we might want to consider adding guards here to avoid the effects if it is already stopped/running

Reply:
I added tests for this